crates/runtime/src/http/v1.rs

```diff
@@ -992,6 +1025,8 @@ pub(crate) mod nsql {
         Extension(nsql_models): Extension<Arc<RwLock<LLMModelStore>>>,
         Json(payload): Json<Request>,
     ) -> Response {
+        let mut q_trace = QueryHistory::default().results_cache_hit(false); // TODO use query cache
```

Comment:
This might need to be in an issue with a link here if it's planned to be fixed later.

Reply:
Not really, moreso just a note to myself now the cache is ready.